### PR TITLE
Change the logic for getting the release

### PR DESCRIPTION
**Pull request by nklmilojevic**
This PR adds correct logic for fetching the release, for example it was downloading x86_64 build for M1 macs which uses arm64.

### CODE REVIEW

**Comment by nklmilojevic** [comment]
@looztra any chance of merging this soon?

**Comment by looztra** [comment]
> @looztra any chance of merging this soon?

sorry @nklmilojevic , I'm not a user of github notifications :).
Let me check that now

**Comment by nklmilojevic** [comment]
Whoops there is one error, fixing it :)

**Comment by nklmilojevic** [comment]
Should be alright now! :)

File: bin/install [hunk 1]
```diff
@@ -156,8 +158,23 @@ get_platform() {
   get_platform_${g_platform_pattern}
 }
 
+get_cpu() {
+  local machine_hardware_name
+  machine_hardware_name="$(uname -m)"
+
+  case "$machine_hardware_name" in
+    'x86_64') local cpu_type="amd64";;
```
Comment (looztra) [inline bin/install]: does not work on my linux env.

It tries to download <https://github.com/derailed/k9s/releases/download/v0.24.14/k9s_Linux_amd64.tar.gz>
when it should be <https://github.com/derailed/k9s/releases/download/v0.24.14/k9s_Linux_x86_64.tar.gz>

**Review by looztra — CHANGES_REQUESTED** [review]
could you try to keep the initial indentation?
2 spaces instead of 4

`shfmt -i 2`

File: bin/install (hunk 1 above)
Comment (looztra) [reply bin/install]: fixed by your latest commit

**Comment by nklmilojevic** [comment]
Oh, apologies. Autoformat screwed it up :) Should be fine now.

**Comment by nklmilojevic** [comment]
Thank you for merging this!

**Comment by looztra** [comment]
merged, thanks! (and sorry for the delay)

**Pull request merged**